pxd.c

```diff
@@ -1996,41 +2033,6 @@ static ssize_t pxd_minor_show(struct device *dev,
 			(unsigned long long)pxd_dev->minor);
 }
 
-static ssize_t pxd_timeout_show(struct device *dev,
-			 struct device_attribute *attr, char *buf)
-{
-	return sprintf(buf, "%u\n", pxd_timeout_secs);
-}
-
-static ssize_t pxd_timeout_store(struct device *dev, struct device_attribute *attr,
```

Comment:
dont remove any of these... this will break the control plane that will set timeout to zero during upgrades and some particular use cases to speed up removal of kernel module.

The PR should enable setting this under top level pxd device tree instead of through individual pxd block device minor path.